lib/yao/resources/tenant.rb

```diff
@@ -5,5 +5,21 @@ class Tenant < Base
     self.service        = "identity"
     self.resource_name  = "tenant"
     self.resources_name = "tenants"
+    self.admin          = true
+
+    class << self
+      def get_by_name(name)
+        self.get("", name: name)
+      end
+      alias find_by_name get_by_name
+
+      def accessible
+        @admin = false
+        tenants = self.list
+        @admin = true
```

Comment:
`admin_disabled` and `as_normal_user` came up to my mind, but I think `as_member` is the best one.